### PR TITLE
Mobile layout

### PR DESCRIPTION
**Pull request by ksmola**
First step towards being more mobile friendly...

Screenshot_20200907-223635_Firefox

### CODE REVIEW

**Review by rusefillc — COMMENTED** [review]
fyi @andreika-git

File: src/view/header.php [hunk 1]
```diff
@@ -28,143 +30,155 @@
...
-		</div>
-		<output id="fileList"></output>
-		<div id="engineInfo" style="display:none;">
-			<fieldset>
-				<legend>Vehicle Information from your Tune file:</legend>
-				<div class="formDiv">
-					<label for="name">Vehicle Name:</label>
-					<input id="name" name="name" type="text" placeholder="" maxlength="32" style="width:10em;" readonly required/>
-				</div>
+		<!-- <span>
```
Comment (andreika-git) [inline src/view/header.php]: Why do we need this span (commented out)?

**Comment by rusefillc** [comment]
@ksmola poke - @andreika-git is waiting for the span clarification

**Comment by ksmola** [comment]
What span are we talking about? I can't see his comment?

**Comment by rusefillc** [comment]
@ksmola 

image

**Comment by rusefillc** [comment]
@andreika-git I wonder if that's a "pending" comment which you have not published yet? you need to finish review for the comment to be visible or something?

**Comment by ksmola** [comment]
Yeah i cant see that. Either way the span can get removed must be some testing/old code relict.

**Review by andreika-git — CHANGES_REQUESTED** [review]

**Pull request merged**